### PR TITLE
OT Frontend Changes - Stage 1

### PR DESCRIPTION
**Pull request by Varun-Sethu**
PR adds the first stage of the required changes to the front-end in order to support operational transform. This PR captures SlateJS operations and propagates them to a central place for delivery to the server. The PR after this will focus on translating SlateJS operations into the CMS operation format + sending them to the server and the final PR will focus on applying operations sent from the server to the editor.

### CODE REVIEW

**Comment by Varun-Sethu** [comment]
Tests are failing because there appears to be something wrong with PIP implying the DB startup script couldn't be run and consequentially the backend testing env couldn't be created

**Review by zijizhu — APPROVED** [review]
Looks good to me!

**Review by lauraw0 — APPROVED** [review]
lgtm! :)

File: postgres/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,7 @@
-FROM python:slim
+FROM python:3.6-alpine
```
Comment (lhjt) [inline postgres/Dockerfile:1]: Any reason why it must be this old?

**Review by lhjt — CHANGES_REQUESTED** [review]

File: postgres/Dockerfile (hunk 1 above)
Comment (Varun-Sethu) [reply postgres/Dockerfile]: should be fixed now

**Review by lhjt — APPROVED** [review]

**Pull request merged**